### PR TITLE
Flesh to stone doesn't make targets invisible.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/tgstation/issues/24998.

Basically: Humans don't use icon_state anymore, so there is no icon_state for the petrified statue, making it invisible. I know the lizard check is terrible but I can't think of a non-horrific way to do it

### CODE REVIEW

**Comment by Incoming5643** [comment]
I hate to break it to you but there are more races than lizard and human_male, and they have a higher than average chance of appearing in wizard rounds.

File: code/game/objects/structures/crates_lockers/closets/statue.dm [hunk 1]
```diff
@@ -82,10 +82,12 @@
 	var/obj/structure/statue/petrified/S = new(loc, src, statue_timer)
 	S.name = "statue of [name]"
 	bleedsuppress = 1
-	S.icon = icon
-	S.icon_state = icon_state
+	if(dna.species.id == "lizard")
+		S.icon_state = "lizard"
+	else
+		S.icon_state = "human_male"
 	S.copy_overlays(overlays)
```
Comment (AnturK) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: Actual error is copy overlays takes atom not overlays lists as a parameter

**Comment by bgobandit** [comment]
lizard is the only race that is likely to have features that don't show up on a statue

File: code/game/objects/structures/crates_lockers/closets/statue.dm (hunk 1 above)
Comment (bgobandit) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: I'm not seeing how this is the actual error, considering the object looks for an icon_state, and there is no "caucasian_m" under mob/human.dmi, hence invisible.

Comment (AnturK) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: It should be copying human overlays on top of it aka actual human here but instead it tries to access overlays.overlays and does nothing instead.

Comment (bgobandit) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: Just tested this -- it does produce a human sprite but it does not copy mutant bodyparts, so lizards would still look wrong

Comment (bgobandit) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: Left was human, right was lizard (ignore the colors that's in the code elsewhere) http://i.imgur.com/FhtjHz9.png

**Review by Cyberboss — CHANGES_REQUESTED** [review]
@RemieRichards fixed it the right way

**Comment by RemieRichards** [comment]
Yeah, I'm da bess

**Pull request closed without merging**